src/blenderbim/blenderbim/bim/module/model/array.py

```diff
@@ -142,8 +142,13 @@ class RemoveArray(bpy.types.Operator, tool.Ifc.Operator):
     bl_label = "Remove Array"
     bl_options = {"REGISTER", "UNDO"}
     item: bpy.props.IntProperty()
+    total_items: bpy.props.IntProperty()
```

Comment:
Don't pass total_items into the operator. It should work it out itself.

Reply:
I made a new commit. Let me know if it solves the issue.